app/assets/stylesheets/views/_landing_page/card.scss

```diff
@@ -0,0 +1,44 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-b-card {
```

Comment:
Currently the cards collide with each other as they have no bottom margin, suggest `margin-bottom: govuk-spacing(6);`

<img width="725" alt="Screenshot 2024-10-10 at 17 41 28" src="https://github.com/user-attachments/assets/b206bead-0e4b-47af-81e0-9b203ca20b3c">

Reply:
Oops, that was an oversight, now added.